internal/servers/controller/handlers/hosts/host_service_test.go

```diff
@@ -320,12 +326,20 @@ func TestList_Static(t *testing.T) {
 				return
 			}
 			require.NoError(gErr)
+			gItems := got.GetItems()
+			sort.Slice(gItems, func(i int, j int) bool {
+				return gItems[i].GetId() < gItems[j].GetId()
+			})
 			assert.Empty(cmp.Diff(got, tc.res, protocmp.Transform()), "ListHosts(%q) got response %q, wanted %q", tc.req, got, tc.res)
 
 			// Test anonymous listing
 			got, gErr = s.ListHosts(auth.DisabledAuthTestContext(iamRepoFn, proj.GetPublicId(), auth.WithUserId(auth.AnonymousUserId)), tc.req)
 			require.NoError(gErr)
 			assert.Len(got.Items, len(tc.res.Items))
+			gItems = got.GetItems()
+			sort.Slice(gItems, func(i int, j int) bool {
+				return gItems[i].GetId() < gItems[j].GetId()
+			})
```

Comment:
This sorting doesn't seem useful.  Please remove.

Reply:
yep seems like the `group by` in the view already covers this sorting

Reply:
I meant it wasn't useful because there isn't anything following this sort which relies on these values being sorted.

Reply:
I put in a sort.Slice on 274 which sorted the `wantHS` and passed the "List Many Hosts" test that was previously failing